### ISSUE TITLE
`pkg.name` can be undefined

### ISSUE THREAD

**Issue by vixalien** [gjsify/ts-for-gir#220]
## Description

The current type of `pkg.name` can be undefined, but it's unclear how common that is. I have to use `pkg.name!` everywhere. This is also true for other types like `pkg.version`

## Current vs Expected Behavior

**Current:**
```typescript
    /** The base name of the entry point (eg. org.foo.Bar.App) */
    export const name: string | undefined;

```

**Expected:**
```typescript
    /** The base name of the entry point (eg. org.foo.Bar.App) */
    export const name: string;
```

## Example

```typescript
pkg.name
```

## References

- Documentation: 
- Source Code: 
- Related Issues: 
- GJS Version: 1.80.2

## Additional Context

- [ ] This affects other GJS types/classes
- [ ] This is a regression from a previous version
- [ ] This blocks other functionality

**Comment by vixalien** [comment]
Also, the same should be there for `gettext()`, `_()`.

**Comment by JumpLink** [comment]
@vixalien As far as I know pkg.name and pkg.version are only defined if pkg.init() was called before. Accordingly, they can be undefined and using ! is a valid way to indicate that you are sure that it is defined.

**Comment by vixalien** [comment]
Sure, it's just a bit inconvenient to always add the !

Funnily enough though, pkg will not always be defined either, so that also needs to be nullable, or, if you can bear with me:

there should be a way to include a file that will mean that developers are sure `pkg.name` will be defined. Having them as nullable really makes it harder.

**Comment by BrainBlasted** [comment]
I agree that having them as nullable makes it harder. I think this is one of the cases where it's OK to let a runtime error happen.
